spec/routing/beta_redirect_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe "Redirecting beta-getintoteaching.education.gov.uk to getintoteaching.education.gov.uk", type: "routing" do
+  before { allow(Rails.configuration.x).to receive(:enable_beta_redirects).and_return(true) }
+
+  specify "redirects content pages" do
```

Comment:
Is this actually checking that it redirects or just that its routing to the correct controller/action? I would have tested this with a request spec that checks it `redirects_to`?

Reply:
It was a bit of a pain to test, this is just ensuring the path remains correct and we're relying on Rails's `#redirect` method to work properly. I tested quite a bit locally by setting up the domains in `/etc/hosts` and it all works as expected.

I'll have a look at doing it via a request spec 👍🏽